library/src/compute/calc.rs

```diff
@@ -131,6 +131,43 @@ pub fn pow(
     result.value()
 }
 
+/// Raise a value to some exponent of e.
+///
+/// ## Example { #example }
+/// ```example
+/// #calc.exp(3)
+/// ```
+///
+/// Display: Power
+/// Category: calculate
+/// Returns: integer or float
+#[func]
+pub fn exp(
+    /// The exponent of the power. Must be non-negative.
```

Comment:
```suggestion
    /// The exponent of the power.
```
`exp` of negative values exists.